app/views/idv/doc_auth/link_sent.html.erb

```diff
@@ -21,7 +21,7 @@
   </div>
   <div class='sm-col sm-col-9 padding-x-1 h2 margin-bottom-2 margin-top-0 margin-y-0'>
     <p><%= t('doc_auth.info.link_sent') %></p>
-    <%= render 'shared/alert', type: 'warning' do %>
+    <%= render AlertComponent.new(type: 'warning') do %>
```

Comment:
we're doing all symbols now right?
```suggestion
    <%= render AlertComponent.new(type: :warning) do %>
```

Reply:
> we're doing all symbols now right?

Yeah, oops 😞   This is what happens with a hurried pre-weekend pull request 😅

Reply:
> we're doing all symbols now right?

Fixed in 2eb846e.